### PR TITLE
Chains/zksync

### PR DESCRIPTION
**Pull request by mshojaei-txfusion**
### Description

- Added support for **Sophon Sepolia Testnet** integration.
- Added support for **ZKSync Sepolia Testnet** integration.

### Backward compatibility

compatible

### Testing

Tested with cli

### CODE REVIEW

**Comment by JaunePomme** [comment]
Changes required imo:
- Changeset is missing,
- Need to run the prettier, 
- Need to fetch the canonical main last changes too,
- For both metadata file, the apiUrls are incorrect I think,
- For both metadata file, the family name, I'm not sure it will be accepted (should be blockscout or other If I remember, someone will tell you)
- For both address file, the file isn't alphabetically correctly ordered (merkleTree at the end of each file),
- I see some width/height for the svg, not sure it's necessary, minimify them both with https://jakearchibald.github.io/svgomg/

File: chains/zksyncsepolia/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://explorer.sepolia.era.zksync.dev/contract_verification 
+    family: zksync
```
Comment (paulbalaji) [inline chains/zksyncsepolia/metadata.yaml:4]: `zksync` is not a supported explorer family type yet
```suggestion
    family: etherscan
```

File: chains/zksyncsepolia/metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    name: zksyncsepoliaexplorer
```
Comment (paulbalaji) [inline chains/zksyncsepolia/metadata.yaml:5]:
```suggestion
    name: zkSync Sepolia Explorer
```

File: chains/zksyncsepolia/metadata.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    url: https://sepolia.explorer.zksync.io/
```
Comment (paulbalaji) [inline chains/zksyncsepolia/metadata.yaml:6]:
```suggestion
    url: https://sepolia.explorer.zksync.io
```

File: chains/zksyncsepolia/metadata.yaml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+chainId: 300
+displayName: ZKSyncSepolia
```
Comment (paulbalaji) [inline chains/zksyncsepolia/metadata.yaml:8]:
```suggestion
displayName: zkSync Sepolia Testnet
```

File: chains/sophonsepolia/metadata.yaml [hunk 5]
```diff
@@ -0,0 +1,19 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://api-explorer-verify.testnet.sophon.xyz/contract_verification
+    family: zksync
+    name: sophonexplorer
```
Comment (paulbalaji) [inline chains/sophonsepolia/metadata.yaml:5]:
```suggestion
    name: Sophon Testnet Explorer
```

File: chains/sophonsepolia/metadata.yaml [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+    url: https://explorer.testnet.sophon.xyz/
```
Comment (paulbalaji) [inline chains/sophonsepolia/metadata.yaml:6]:
```suggestion
    url: https://explorer.testnet.sophon.xyz
```

File: chains/sophonsepolia/metadata.yaml [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+chainId: 531050104
+displayName: SophonSepolia
```
Comment (paulbalaji) [inline chains/sophonsepolia/metadata.yaml:8]:
```suggestion
displayName: Sophon Sepolia
```

File: chains/sophonsepolia/metadata.yaml [hunk 8]
```diff
@@ -0,0 +1,19 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://api-explorer-verify.testnet.sophon.xyz/contract_verification
+    family: zksync
```
Comment (paulbalaji) [inline chains/sophonsepolia/metadata.yaml:4]: `zksync` explorer type is not valid schema yet
```suggestion
    family: etherscan
```

**Review by paulbalaji — COMMENTED** [review]
please also add a minor changeset with `yarn changeset add`, and run `yarn lint:fix` and `yarn prettier`

File: chains/sophonsepolia/metadata.yaml [hunk 9]
```diff
@@ -0,0 +1,19 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://api-explorer-verify.testnet.sophon.xyz/contract_verification
+    family: etherscan
+    name: Sophon Testnet Explorer
+    url: https://explorer.testnet.sophon.xyz
+chainId: 531050104
```
Comment (paulbalaji) [inline chains/sophonsepolia/metadata.yaml:7]: add deployer info
```suggestion
chainId: 531050104
deployer:
  name: NAME
  url: LINK
```

File: chains/zksyncsepolia/metadata.yaml [hunk 10]
```diff
@@ -0,0 +1,19 @@
+# yaml-language-server: $schema=../schema.json
+blockExplorers:
+  - apiUrl: https://explorer.sepolia.era.zksync.dev/contract_verification
+    family: etherscan
+    name: zkSync Sepolia Explorer
+    url: https://sepolia.explorer.zksync.io
+chainId: 300
```
Comment (paulbalaji) [inline chains/zksyncsepolia/metadata.yaml:7]: add deployer info
```suggestion
chainId: 300
deployer:
  name: NAME
  url: LINK
```

**Review by paulbalaji — COMMENTED** [review]
looks great! just need to add the deployer metadata and we're good to merge

File: chains/sophonsepolia/metadata.yaml (hunk 9 above)
Comment (mshojaei-txfusion) [reply chains/sophonsepolia/metadata.yaml:7]: Is this what we should put there?

deployer:
  name: Abacus Works
  url: https://www.hyperlane.xyz

Comment (paulbalaji) [reply chains/sophonsepolia/metadata.yaml:7]: with tx fusion deployer details

**Review by paulbalaji — APPROVED** [review]

**Pull request merged**